Packs/Jira/Integrations/JiraV3/README.md

```diff
@@ -8,7 +8,7 @@ If you are upgrading from a previous version of this integration, see [Breaking
 
 1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
 2. Search for Atlassian Jira V3.
-3. **Authentication**: Basic Authentication and OAuth 2.0 are used for both Jira Cloud, and OnPrem. Read the [Authentication](#authentication) process in order to configure your instance
+3. **Authentication**: Basic Authentication, Personal Access Token and OAuth 2.0 are used for both Jira Cloud, and OnPrem. Read the [Authentication](#authentication) process in order to configure your instance
```

Comment:
Basic Authentication, Personal Access Token, and OAuth 2.0 are used for Jira Cloud and On-prem...